integration/test/draining/draining_test.go

```diff
@@ -0,0 +1,249 @@
+// +build k8srequired
+
+package draining
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/giantswarm/apprclient"
+	"github.com/giantswarm/e2e-harness/pkg/framework"
+	"github.com/giantswarm/helmclient"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/spf13/afero"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/rest"
+
+	"github.com/giantswarm/aws-operator/integration/env"
+	"github.com/giantswarm/aws-operator/service/awsconfig/v10/key"
+)
+
+const (
+	CNRAddress      = "https://quay.io"
+	CNROrganization = "giantswarm"
+	ChartChannel    = "stable"
+	ChartName       = "e2e-app-chart"
+	ChartNamespace  = "e2e-app"
+)
+
+func Test_Draining(t *testing.T) {
```

Comment:
This is the draining test.

Reply:
It was difficult for me to anticipate what this test is about. I think a comment would helpful. Something like this tests ensures e2e-app running on the draining node is up for more than 95% of the time.